assets/app/features/upgrade/controllers/upgrade-nodes.controller.js

```diff
@@ -81,6 +122,21 @@
             if (response.data.errors) {
                 return;
             }
+
+            // init upgrade compute checkbox
+            if (vm.nodesUpgrade.upgradeComputeCheckboxSelected === null) {
```

Comment:
Why not just default to `true` and override it based on the `response`?

Reply:
will do